.github/workflows/etcd-defrag.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish
+
+on:
+- push
+- workflow_dispatch
+
+jobs:
+
+  publish-tagged:
+    if: |
+      startsWith(github.ref, 'refs/tags/v') &&
+      !github.event.pull_request.head.repo.fork
```

Comment:
If I understand it correctly, the workflow is triggered on each push event, but it will not execute if the latest commit isn't tagged. Correct?

Reply:
correct, I use that to differentiate between development builds (nightlies) and tagged release.